app/models/page/DashboardPage.scala

```diff
@@ -107,7 +107,11 @@ object DashboardPage {
 
   private def activeApplication(user: CachedData)(implicit request: RequestHeader, lang: Lang): DashboardPage = {
     val firstStep = if (RoleUtils.activeUserWithApp(user)) ProgressActive else ProgressInactive
-    val secondStep = if (DisplayOnlineTestSectionRole.isAuthorized(user)) ProgressActive else ProgressInactive
+    val secondStep = if (DisplayOnlineTestSectionRole.isAuthorized(user) || FastPassReceivedRole.isAuthorized(user)) {
```

Comment:
I think we are using the roles in many situations:
-business logic
-to control what we display in the screen
-to control access

I think there are 3 different problems with different needs.

We have the risk of making everything dependent on the roles and then it will be too risky to change them, so we will create many more.